Dockerfile.deploy

```diff
@@ -2,11 +2,12 @@ FROM alpine as artifacts
 
 RUN apk add git maven openjdk8 bash zip curl -f
 
-COPY ./demo_processes /usr/src/demo_processes
-COPY ./org            /usr/src/org
-COPY ./src            /usr/src/src
-COPY ./seed           /usr/src/seed
-COPY ./pom.xml        /usr/src/pom.xml
+COPY ./demo_processes/pizza_order/src     /usr/src/demo_processes/pizza_order/src
+COPY ./demo_processes/pizza_order/pom.xml /usr/src/demo_processes/pizza_order/pom.xml
```

Comment:
You're using build.sh script few lines below but don't copy it into an image. It'll fail.

Reply:
This task was in development, again. Thank you for your attention, cool 😮

Reply:
If it is still in development, then add WIP into the pull request name and don't change status in Jira to Awaiting review.